pliers/extractors/audio.py

```diff
@@ -258,6 +316,56 @@ class RMSEExtractor(LibrosaFeatureExtractor):
 
     _feature = 'rmse'
 
+class RMSExtractor(LibrosaFeatureExtractor):
+
+    ''' Extracts root mean square (RMS) from audio using the Librosa
+    library.
+
+    For details on argument specification visit:
+    https://librosa.github.io/librosa/feature.html.'''
+
+    _feature = 'rms'
```

Comment:
It looks like RMS is just renamed from RMSE and RMSE will be removed in librosa `0.7.0`, https://github.com/librosa/librosa/blob/master/librosa/feature/spectral.py#L833

I'd rather just support one (RMS), and delete the code for RMSE. We will have to update `optional-dependencies.txt` to have `librosa>=0.6.3`

Reply:
+1